src/vs/code/electron-main/windows.ts

```diff
@@ -761,6 +768,8 @@ export class WindowsManager implements IWindowsMainService {
 			}
 		}
 
+		this.backupService.registerWindowForBackups(vscodeWindow.id, !configuration.workspacePath, configuration.backupFolder, configuration.workspacePath);
```

Comment:
@Tyriar are you missing the check for `openConfig.cli.extensionDevelopmentPath` on purpose here?